mesheryctl.sh

```diff
@@ -20,50 +27,70 @@ main() {
 	local service_mesh=
 	local test_name=
 	local load_generator=
+	local mesh_deployed=
+	local PLATFORM=docker
+
+	# load balancer
+	minikube tunnel &> /dev/null &
+	sleep 10
 
 	parse_command_line "$@"
 
 	# perform the test given in the provided profile_id
 	if [ -z "$perf_profile_name" ]
 	then
-		for mesh in "${!adapters[@]}"
-		do
-			shortName=$(echo ${adapters["$mesh"]} | cut -d '-' -f2 | cut -d ':' -f1)
 
-			docker network connect bridge meshery_meshery-"$shortName"_1
-			docker network connect minikube meshery_meshery-"$shortName"_1
-		done
+		# connect containers and deploy service mesh
+		if [[ $PLATFORM == "docker" ]]
+		then
+			shortName=$(echo ${adapters["$service_mesh"]} | cut -d '-' -f2 | cut -d ':' -f1)
+			docker_networking "$shortName"
+		fi
 
-		docker network connect bridge meshery_meshery_1
-		docker network connect minikube meshery_meshery_1
-		mesheryctl system config minikube -t ~/auth.json
+		if [[ $mesh_deployed != "true" ]]
+		then
+			deploy_mesh "$service_mesh" "$PLATFORM"
+		fi
+
+		kubectl get pods --all-namespaces
 
 		echo "Configuration file: $perf_filename"
 		echo "Endpoint URL: $endpoint_url"
 		echo "Service Mesh: $service_mesh"
 		echo "Test Name: $test_name"
 		echo "Load Generator: $load_generator"
 
-		mesheryctl perf apply --file $GITHUB_WORKSPACE/.github/$perf_filename -t ~/auth.json --url "$endpoint_url" --mesh "$service_mesh" --name "$test_name" --load-generator "$load_generator"
+		mesheryctl perf apply --file $GITHUB_WORKSPACE/.github/$perf_filename -t ~/auth.json --url "$endpoint_url" --mesh "$service_mesh" --name "$test_name" --load-generator "$load_generator" test-profile
```

Comment:
Can we name profile  with some iconic characters like date, time.. instead of `test-profile`?

Reply:
@gyohuangxin We might move away from this PR and rewrite this as there has been a lot more changes in the current master branch.

Currently we take the profile name as an input in the action. And in the scheduled workflows, we use contextual profile names.

Reply:
@navendu-pottekkat Got it, thanks!